### PR TITLE
EZP-32395: Sort out Locations tab info on visibility

### PR DESCRIPTION
**Pull request by barw4**
| Question      | Answer
| ------------- | ---
| Tickets       | https://issues.ibexa.co/browse/EZP-32395
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Tests pass?   | yes
| Doc needed?   | no
| License       | [GPL-2.0](https://github.com/ezsystems/ezplatform-admin-ui/blob/master/LICENSE)

`supplyIsLocationInvisible` method is not really necessary, we can easily check if the `Location` is hidden in the twig itself + it shows invalid data as it only accounts for a single `Location`.  A small performance increase is an upside as well.

#### Checklist:
- [x] Coding standards (`$ composer fix-cs`)
- [x] Ready for Code Review

### CODE REVIEW

File: src/bundle/Resources/views/content/locationview.html.twig [hunk 1]
```diff
@@ -73,7 +73,6 @@
                             'system_urls_pagination_params': system_urls_pagination_params,
                             'roles_pagination_params': roles_pagination_params,
                             'policies_pagination_params': policies_pagination_params,
-                            'is_location_visible': is_location_visible,
```
Comment (adamwojs) [inline src/bundle/Resources/views/content/locationview.html.twig:76]: Technically this is BC break, so let leave this variable (with deprecation node) and pass here location.invisible

File: src/bundle/Resources/views/content/tab/locations/tab.html.twig [hunk 2]
```diff
@@ -51,7 +51,7 @@
                                     {% if not can_hide[location.id] %} disabled="disabled"{% endif %}
                                     value="{{ location.id }}"/>
                             </label>
-                            {% if location.contentInfo.isHidden or not is_location_visible %}
+                            {% if location.invisible or location.hidden %}
```
Comment (adamwojs) [inline src/bundle/Resources/views/content/tab/locations/tab.html.twig:54]: According to https://github.com/ezsystems/ezplatform-kernel/blob/94937f8db831eb4229686f6cfde92e052a658b11/eZ/Publish/API/Repository/Values/Content/Location.php#L23:
> Indicates that the Location is not visible, being either marked as hidden itself, or implicitly hidden by its Content or an ancestor Location

`location.invisible` should be enough :wink:

**Review by adamwojs — APPROVED** [review]

**Review by damianz5 — APPROVED** [review]

**Review by tomaszszopinski — APPROVED** [review]
QA approved on eZPlatform 2.5 with patch.

**Comment by lserwatka** [comment]
Could you merge it up?

**Comment by barw4** [comment]
Merged:
into `2.2`: https://github.com/ezsystems/ezplatform-admin-ui/commit/3b8389856a40ea094a7a912d2e53c827e2ff19b3
into `master`: https://github.com/ezsystems/ezplatform-admin-ui/commit/40d47bb8d4af8287491b543eac7113ea56342478

**Pull request merged**